From 377fc39ad4d33a813c823fa73a9c10d7c2e8a2fe Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Fri, 1 Sep 2017 13:38:14 +0800
Subject: [PATCH 2482/5242] MLK-16350-2: ASoC: fsl_mqs: enable the pm_runtime

commit  d113d055566e0600e9535429ee2e13d9dbf1b002 from
https://source.codeaurora.org/external/imx/linux-imx.git

enable the pm_runtime, the power domains wil be disabled when
mqs is idle.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/codecs/fsl_mqs.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/codecs/fsl_mqs.c b/sound/soc/codecs/fsl_mqs.c
index e00aa2f..0a0335e 100644
--- a/sound/soc/codecs/fsl_mqs.c
+++ b/sound/soc/codecs/fsl_mqs.c
@@ -13,6 +13,7 @@
 #include <linux/moduleparam.h>
 #include <linux/mfd/syscon.h>
 #include <linux/mfd/syscon/imx6q-iomuxc-gpr.h>
+#include <linux/pm_runtime.h>
 #include <linux/of.h>
 #include <linux/pm.h>
 #include <linux/slab.h>
@@ -277,6 +278,7 @@ static int fsl_mqs_probe(struct platform_device *pdev)
 	}
 
 	dev_set_drvdata(&pdev->dev, mqs_priv);
+	pm_runtime_enable(&pdev->dev);
 
 	return devm_snd_soc_register_component(&pdev->dev, &soc_component_fsl_mqs,
 			&fsl_mqs_dai, 1);
-- 
1.7.9.5

